contrib/examples/apiserver/instance.yaml

```diff
@@ -3,4 +3,6 @@ kind: Instance
 metadata:
   name: test-instance
 spec:
-  serviceClassName: dugs awesome service instance
+  serviceClassName: test-serviceclass
```

Comment:
@pmorie can we work @duglin in somewhere? May he forever live in the glory of the service catalog!